python-package/lightgbm/callback.py

```diff
@@ -473,4 +477,6 @@ def early_stopping(stopping_rounds: int, first_metric_only: bool = False, verbos
     callback : _EarlyStoppingCallback
         The callback that activates early stopping.
     """
-    return _EarlyStoppingCallback(stopping_rounds=stopping_rounds, first_metric_only=first_metric_only, verbose=verbose, min_delta=min_delta)
+    return _EarlyStoppingCallback(
+        stopping_rounds=stopping_rounds, first_metric_only=first_metric_only, verbose=verbose, min_delta=min_delta
+    )
```

Comment:
Should we also add the trailing comma here?

Reply:
updated in https://github.com/microsoft/LightGBM/pull/6336/commits/25631586bc4f13782d9b2622e347e7c8d37a0aa3